### PR TITLE
[ALLUXIO-2593] Fix logging style in AbstractThriftClient 

### PR DESCRIPTION
**Pull request by wanghk00a1**
https://alluxio.atlassian.net/browse/ALLUXIO-2593
Replace all occurrences of
LOG.error("Failed after " + retryPolicy.getRetryCount() + " retries.");
with
LOG.error("Failed after {} retries.", retryPolicy.getRetryCount());

### CODE REVIEW

**Comment by wanghk00a1** [comment]
change title from ALLUXIO­2593 to ALLUXIO­-2593

**Comment by wanghk00a1** [comment]
alluxio-bot, check this please

**Comment by wanghk00a1** [comment]
alluxio-bot, check this please

**Comment by wanghk00a1** [comment]
alluxio-bot, check this please

**Comment by wanghk00a1** [comment]
alluxio-bot, check this please

**Comment by ifcharming** [comment]
ok to test

**Comment by ifcharming** [comment]
add to whitelist

File: core/common/src/main/java/alluxio/AbstractThriftClient.java [hunk 1]
```diff
@@ -165,7 +166,8 @@
       }
     } while (retryPolicy.attemptRetry());
 
-    LOG.error("Failed after " + retryPolicy.getRetryCount() + " retries.");
+    //LOG.error("Failed after " + retryPolicy.getRetryCount() + " retries.");
+    LOG.error("Failed after {} retries.", retryPolicy.getRetryCount());
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/AbstractThriftClient.java]: We already have git for version control, so there is no need to comment out the modified code, could you please delete it?

Comment (wanghk00a1) [inline core/common/src/main/java/alluxio/AbstractThriftClient.java]: I'm sorry about it. I have deleted the modified code now. Please check this. Thanks.

**Comment by wanghk00a1** [comment]
alluxio-bot, check this please

**Comment by wanghk00a1** [comment]
alluxio-bot, check this please

**Comment by aaudiber** [comment]
Hey @wanghk00a1, can you sign the license agreement? The link is [here](http://cla.alluxio.org/)

**Comment by calvinjia** [comment]
@wanghk00a1 Thanks for improving the code

**Pull request merged**